ios/xcconfig/Identity.xcconfig

```diff
@@ -0,0 +1,8 @@
+// Change to yours (you find the id by 'security find-identity -v' it is the one inside the brackets, )
+// and then make git ignore it by
+// git update-index --skip-worktree BuildConfig/Identity.xcconfig
+DEVELOPMENT_TEAM = FC967L3QRG
+CODE_SIGN_IDENTITY = Mac Developer
```

Comment:
Shouldn't this be Apple Developer?

Reply:
Judging from your existing project, Apple Development should be the right thing. You probably have to double check this with your CI though. I changed it to that for now.

Reply:
It might also be that you need it to be `iPhone Developer`. But interestingly for me locally this all made no difference currently.